app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/rules/ExceptionRulesDebugReceiver.kt

```diff
@@ -111,17 +108,14 @@ class ExceptionRulesDebugReceiverRegister @Inject constructor(
         coroutineScope: CoroutineScope,
         vpnStopReason: VpnStopReason,
     ) {
-        // only for debug builds
-        if (appBuildConfig.isDebug) {
```

Comment:
Removed condition as this functionality should be the same in debug and release. This is only used in internal builds. The rest of the code in this function is untouched.